program/src/lib.rs

```diff
@@ -1,7 +1,5 @@
 //! A program to store metadata information for programs.
 
-#![no_std]
-
```

Comment:
Just checking this is intentional.

Reply:
Yeah, we need separate the interface to make it work properly.